lib/manager/actions/status.js

```diff
@@ -1,12 +1,13 @@
 // @flow
 
+import * as React from 'react'
```

Comment:
```suggestion
import { Node } from 'react'
```
with appropriate change in the type that uses it.

Reply:
Hmm, I don't think that's going to work because React needs to be imported for the jsx that is handled in this file.

Reply:
Sorry, how about `import React, { Node } from 'react'`?